ecl/eclcc/eclcc.hpp

```diff
@@ -96,6 +96,7 @@ const char * const helpText[] = {
     "!   -pch          Generate precompiled header for eclinclude4.hpp",
 #endif
     "!   -P <path>     Specify the path of the output files (only with -b option)",
+    "!   --releasemem  Fully clean up memory (useful when checking for memory leaks)",
```

Comment:
I feel that this description is confusing when not knowing its implementation. From the top, 'releasemem' could imply that this is an assertion, an option to make sure that all mem is actually released rather than used to check that the release code does actually release all mem when used in conjunction with a leak tester. I fell that we should state that the code exits letting the OS clean up the mem and that using this op prevents that and cleans up the mem itself.